Code/Tools/ProjectManager/Source/UpdateProjectCtrl.cpp

```diff
@@ -94,6 +94,16 @@ namespace O3DE::ProjectManager
         return ProjectManagerScreen::UpdateProject;
     }
 
+    bool UpdateProjectCtrl::ContainsScreen(ProjectManagerScreen screen)
+    {
+        return screen == GetScreenEnum() || screen == ProjectManagerScreen::GemCatalog;
```

Comment:
should this also check GemRepos?

Reply:
Yea that's the point of leaving it out. We don't want to advertise jumping to it from any other screen here. I'll add a comment about that though.